tests/fixtures/disallow-multiple-var-decl/good/for.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable ember-suave/prefer-destructuring */
```

Comment:
Yeah, I would expect all files to adhere to all other rules, except the one they are testing.